datumaro/plugins/transforms.py

```diff
@@ -541,4 +541,19 @@ def transform_item(self, item):
                     annotations.append(ann.wrap(label=conv_label))
             else:
                 annotations.append(ann.wrap())
+        return item.wrap(annotations=annotations)
+
+class TransformLabels(Transform, CliPlugin):
```

Comment:
I feel that we need to find a more self-explanatory name. I suggest `AnnsToLabels` or `CollectLabels`. Think of `datum transform [how]`.